views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
I was trying to do the full expansion of the matrix multiplications and it seems that using Bernstein's form comes pretty straightforward (see http://mathb.in/42039 for left-associative and http://mathb.in/42040 for right associative).

How did you do the expansion that led you to the notation with (z-1) instead of (1-z)? I can start from there for preserving your approach and add the sign-change at the end like you suggested,  or I can just use one of the expansions linked above (but it's your call).